ios/Video/ReactNativeVideoManager.swift

```diff
@@ -6,31 +6,33 @@
 import Foundation
 
 public class ReactNativeVideoManager: RNVPlugin {
-    private let expectedMaxVideoCount = 2
+    private let expectedMaxVideoCount = 10
 
     // create a private initializer
     private init() {}
 
     public static let shared: ReactNativeVideoManager = .init()
 
-    private var instanceCount = 0
-    private var pluginList: [RNVPlugin] = Array()
+    var instanceList: [RCTVideo] = Array()
```

Comment:
This file has already been reworked on master branch, can you revert these changes (It cause memory leak)